securedrop/source_templates/index.html

```diff
@@ -14,7 +14,7 @@
     {% endassets %}
   </head>
   <body>
-    <div class="js-warning warning">{{ gettext('<strong>We recommend turning the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
+    <div class="js-warning warning">{{ gettext('<strong>We recommend moving the Security Slider to Safest to protect your anonymity:</strong> <a id="disable-js" href="">Learn how to set it to safest</a>, or ignore this warning to continue.') }} <img id="js-warning-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
     <div class="use-tor-browser warning">{{ gettext('<strong>We recommend using Tor Browser to access SecureDrop:</strong> <a id="recommend-tor" href="{tor_browser_url}">Learn how to install it</a>, or ignore this warning to continue.').format(tor_browser_url=url_for('info.recommend_tor_browser')) }} <img id="use-tor-browser-close" src="{{ url_for('static', filename='i/font-awesome/times-white.png') }}" width="12px" height="12px"></div>
```

Comment:
Suggesting: "It is recommended" or "* recommends"

Reply:
done